### DISCUSSION TITLE
How to check the speed and power consumption?

### DISCUSSION THREAD

**Discussion by ngdymx** [AMDResearch/Riallto#20] [Q&A] [answered]
Hi,

Can we check the efficiency like the speed and power consumption when running the application on NPU, and how to do it?
Thank you very much!

**Comment by STFleming** [comment, marked as answer]
Hi @ngdymx,

Thanks for your question; I don't have a recommended way of doing this. However, in the past, we have seen people using a battery monitor to measure the discharge rate of their laptop batteries while running different workloads. Using a battery monitor is probably not a very accurate technique, but it will give a rough idea. 

All the best,
Shane
